governance/votes/Executive vote - December 10, 2021.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Template - [Executive Vote] Parameter Changes, Switching MKR Vesting Source - December 10, 2021
+summary: Changes to wstETH-A and MATIC-A Parameters, replacing current MKR vesting streams with streams funded by the Protocol Treasury MKR.
+date: 2021-12-10T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Parameter Changes, Switching MKR Vesting Source - December 10, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- wstETH-A changes to [Debt Ceiling Instant Access Module](https://makerdao.world/en/learn/governance/module-dciam) and Liquidation Parameters, as detailed below. 
```

Comment:
I think it's a bit awkward that this item has the form "wstETH-A changes to DC-IAM..." and the next item has the form "MATIC-A DC-IAM changes". I can see why you did it: This item also has Liquidation Parameters.

Maybe "Changes to [...]"? So: "Changes to wstETH-A DC-IAM and [wstETH-A] Liquidation Parameters".